src/content/developers/docs/miner-extractable-value/index.md

```diff
@@ -0,0 +1,123 @@
+---
+title: Miner extractable value
```

Comment:
```suggestion
title: Miner extractable value (MEV)
```

Reply:
How does the title `Maximal (miner) extractable value (MEV)` sound?

Reply:
> How does the title `Maximal (miner) extractable value (MEV)` sound?

I don't think it conveys the point you intend to hit. To me just sounds like it's the maximal extractable value for miners.

Personally, I'd leave it as `Miner extractable value (MEV)` for now and just explain minimal/maximal in the body. If only because most people who know the term `maximal extractable value` kinda also means `miner extractable value` but the opposite isn't true.

Reply:
sounds good, will do